app/views/tag/show/_tab_content.html.erb

```diff
@@ -1,7 +1,8 @@
 <div class="tab-content" style="clear:both;">
 
   <% if @start && @end %>
-    <p><%= raw translation('tag.show.displaying_content') %> <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %>. <a href="/tag/<%=h params[:id] %>">View all</a></p>
+<p><%= raw translation('tag.show.displaying_content') %> <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %>. <a href="/tag/<%=h params[:id] %>"><%= raw translation('tag.show.view_all') %></a></p>
+
```

Comment:
@RishiKumarRay You need to indent line 4 so that it’s aligned with the line that you removed. The new line on line 5 is not needed. Thank you!